tests/models/bart/test_modeling_tf_bart.py

```diff
@@ -230,69 +230,6 @@ def test_model_common_attributes(self):
                 name = model.get_bias()
                 assert name is None
 
-    def test_resize_token_embeddings(self):
```

Comment:
The exact same test exists in `test_modeling_tf_common`, and is touched in this PR